packages/tracing/src/browser/browsertracing.ts

```diff
@@ -61,10 +61,15 @@ export interface BrowserTracingOptions extends RequestInstrumentationOptions {
   markBackgroundTransactions: boolean;
 
   /**
-   * beforeNavigate is called before a pageload/navigation transaction is created and allows for users
-   * to set custom transaction context. Default behavior is to return `window.location.pathname`.
+   * beforeNavigate is called before a pageload/navigation transaction is created and allows users to modify transaction
+   * context data, or drop the transaction entirely (by setting `sampled = false` in the context).
```

Comment:
L:

"drop the transaction entirely" and "`sampled = false`" are not the same behavior.

Reply:
From a user perspective they're the same - the transaction isn't sent - and that's the target audience of the docstring. For the developer reading the docstring, it also tells them _how_ the transaction is dropped, so there's no confusion about whether it gets pulled off of the scope or something.

(It's also is in line with the language we use for error events - when you "drop" the event, it's not that it's not created, it's that it's not sent, and same here.)